build.gradle

```diff
@@ -65,14 +70,27 @@ spotless {
 
 
 // OpenAPI Server Generation
+// - Unzip external dependency on Cloud Resource Schema
 // - specify the source and target
 // - define the generation
 // - define the validation
 // - define dependencies to include the generated code
-
 def openapiSourceFile = "${projectDir}/src/main/resources/api/service_openapi.yaml"
 def openapiTargetDir = "${buildDir}/generated"
 
+configurations {
+    cloudResourceSchema { transitive = false }
+}
+dependencies {
+    cloudResourceSchema group: 'bio.terra.cloud-resource-lib', name: 'cloud-resource-schema', version: '0.0.0-SNAPSHOT'
+}
+task unzipCloudResourceSchema(type: Copy) {
+    from zipTree(configurations.cloudResourceSchema.singleFile).matching {
+        include 'cloud_resources_uid.yaml'
+    }
+    into "$openapiTargetDir"
```

Comment:
I turned to just use the  openapiTargetDir instead puting it under builddir/src/main/resoruce/api